internal/pkg/flink/internal/utils/input.go

```diff
@@ -20,11 +20,12 @@ func GetStdin() *term.State {
 }
 
 func GetConsoleParser() prompt.ConsoleParser {
-	defer func() {
-		if r := recover(); r != nil {
-			log.CliLogger.Warnf("Couldn't open \"/dev/ttv\" file. Error: %v\n", r)
-		}
-	}()
+	if fileInfo, err := os.Stat("/dev/tty"); err != nil {
+		log.CliLogger.Warnf("Couldn't open \"/dev/tty\" file because it doesn't exist. Error: %v\n", err)
+	} else if fileInfo.Mode().Perm()&0444 != 0 {
```

Comment:
Shouldn't this equal 0 if no read permissions are set? Also, why are we checking all three (user, group, other)?

Reply:
It is equal 0 if no read permissions are set. I fixed that in the most recent commit.

My thinking in checking all 3 (user, group, other) is that the CLI could be containerized or on a shared machine where the user of the CLI may not be the owner of the `\dev\tty` file.

Reply:
On docker:
```
f8ae59e12f7d:/$ ls dev/tty -l
crw-rw-rw-    1 root     root        5,   0 Aug 11 22:09 dev/tty
f8ae59e12f7d:/$ whoami
confluent
```